test-inputs/valid-wasm-binaries.txt

```diff
@@ -71,50 +71,90 @@ wasm-feature-tests/small-manually-written/build/select.wat.wasm
+wasm-spec-tests/build/br.wasm
+wasm-spec-tests/build/br_table.wasm
+wasm-spec-tests/build/br.wasm
+wasm-spec-tests/build/bulk.wasm
+wasm-spec-tests/build/bulk.wasm
+wasm-spec-tests/build/call.wasm
 wasm-spec-tests/build/call_indirect.wasm
+wasm-spec-tests/build/call.wasm
+wasm-spec-tests/build/conversions.wasm
 wasm-spec-tests/build/comments.wasm
 wasm-spec-tests/build/const.wasm
+wasm-spec-tests/build/elem.wasm
+wasm-spec-tests/build/conversions.wasm
 wasm-spec-tests/build/custom.wasm
 wasm-spec-tests/build/data.wasm
+wasm-spec-tests/build/elem.wasm
 wasm-spec-tests/build/endianness.wasm
 wasm-spec-tests/build/exports.wasm
 wasm-spec-tests/build/f32_bitwise.wasm
 wasm-spec-tests/build/f32_cmp.wasm
+wasm-spec-tests/build/fac.wasm
 wasm-spec-tests/build/f32.wasm
 wasm-spec-tests/build/f64_bitwise.wasm
 wasm-spec-tests/build/f64_cmp.wasm
 wasm-spec-tests/build/f64.wasm
+wasm-spec-tests/build/fac.wasm
 wasm-spec-tests/build/float_exprs.wasm
 wasm-spec-tests/build/float_literals.wasm
 wasm-spec-tests/build/float_memory.wasm
+wasm-spec-tests/build/i32.wasm
+wasm-spec-tests/build/i64.wasm
+wasm-spec-tests/build/if.wasm
 wasm-spec-tests/build/float_misc.wasm
 wasm-spec-tests/build/forward.wasm
 wasm-spec-tests/build/func_ptrs.wasm
 wasm-spec-tests/build/func.wasm
 wasm-spec-tests/build/global.wasm
+wasm-spec-tests/build/i32.wasm
+wasm-spec-tests/build/i64.wasm
+wasm-spec-tests/build/if.wasm
 wasm-spec-tests/build/imports.wasm
 wasm-spec-tests/build/inline-module.wasm
 wasm-spec-tests/build/int_exprs.wasm
+wasm-spec-tests/build/loop.wasm
+wasm-spec-tests/build/memory_copy.wasm
+wasm-spec-tests/build/memory_fill.wasm
 wasm-spec-tests/build/int_literals.wasm
+wasm-spec-tests/build/memory_init.wasm
 wasm-spec-tests/build/labels.wasm
 wasm-spec-tests/build/left-to-right.wasm
 wasm-spec-tests/build/linking.wasm
 wasm-spec-tests/build/load.wasm
 wasm-spec-tests/build/local_get.wasm
 wasm-spec-tests/build/local_set.wasm
+wasm-spec-tests/build/ref_is_null.wasm
+wasm-spec-tests/build/ref_null.wasm
 wasm-spec-tests/build/local_tee.wasm
+wasm-spec-tests/build/select.wasm
+wasm-spec-tests/build/memory_fill.wasm
 wasm-spec-tests/build/memory_grow.wasm
+wasm-spec-tests/build/memory_init.wasm
 wasm-spec-tests/build/memory_redundancy.wasm
 wasm-spec-tests/build/memory_size.wasm
+wasm-spec-tests/build/table_copy.wasm
+wasm-spec-tests/build/table_fill.wasm
+wasm-spec-tests/build/table_get.wasm
+wasm-spec-tests/build/table_grow.wasm
+wasm-spec-tests/build/table_init.wasm
+wasm-spec-tests/build/table_set.wasm
+wasm-spec-tests/build/table_size.wasm
+wasm-spec-tests/build/table.wasm
 wasm-spec-tests/build/memory_trap.wasm
+wasm-spec-tests/build/type.wasm
 wasm-spec-tests/build/memory.wasm
 wasm-spec-tests/build/names.wasm
 wasm-spec-tests/build/nop.wasm
 wasm-spec-tests/build/return.wasm
-# Uses typed select instruction, which is non-MVP, but it is not filtered out by wasm-validate when disabling extensions.
-# wasm-spec-tests/build/select.wasm
```

Comment:
Can Wasabi instrument typed select now? If yes, we should uncomment this line (i.e., add the test case).

Reply:
I think this is included in [this line](https://github.com/danleh/wasabi/pull/41/files#diff-468180d79cb13ce9e8144e0b2a658548811da2d08197f9f2ea1ee62181c89820R138).